pkg/acpid/build.yml

```diff
@@ -1,5 +1,9 @@
 image: acpid
 config:
   binds:
-  - /dev:/dev
+    - /dev:/dev
+  devices:
+    - path: all
+      type: c
+      major: 13
```

Comment:
could you please explain in the commit message why this fixes it or if there potentially can be a smaller set up devices mapped?

Reply:
Sure, I can amend the commit message. But as you can see in the bug, I'm not really sure which devices needs to be allowed for ACPId. I know that "input" devices have a major number set to 13, but I don't know if ACPId needs to access other types of devices. Anyway, it seems to be ok on my machine, that's why I pushed this change. Even if it's not enough, it's still better than the current state of ACPID configuration.

Reply:
I looked into busybox' acpid code, and it seems that it needs access to /dev/input/event* only. So I guess that allowing all "input" devices is ok. So I amended the relevant commit, should be ok now.